### PR TITLE
contact form pull request

### PR DESCRIPTION
**Pull request by kkasaija**
Modify Html and CSS files.
Change image files from png to SVG.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,724 @@
+body {
+  background-color: #e5e5e5;
+}
+
+.global-wrapper {
+  position: relative;
+  width: 375px;
+  height: 5410px;
```
Comment (mgmediaweb) [inline style.css:8]: Kindly, I suggest you not use a static width, it is better to use percentages at 100% and make your site adapt to any device. just like height, the container better fits the content.

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @Kasaija-Kenneth , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost1

## Highlights 💯 
- Professional-looking readme file. ✔️ 
- Clear and descriptive Pull Request. ✔️ 
- Clean and tidy code. ✔️ 
- Good job following the design. ✔️ 


## Required Changes ♻️

Kindly, I suggest you follow the guidelines of the original Figma design a little more, I indicate them below:

image

Kindly, I suggest you implement effects on the hover events on all buttons, as Figma indicates.
image

image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+            <ul id="details2">
+              <li><a href=""><img src="images/Github.svg" alt="Github"></a></li>
+              <li><a href=""><img src="images/LinkedIn.svg" alt="LinkedIn"></a></li>
+              <li><a href=""><img src="images/Angellist.svg" alt="Angellist"></a></li>
+              <li><a href=""><img src="images/Twitter.svg" alt="Twitter"></a></li>
+              <li><a href=""><img src="images/Medium.svg" alt="Medium"></a></li>
+            </ul>
+          </div>
+          <button class="card-check" type="button">Get my resume</button>
+        </div>
```
Comment (mahbubul14) [inline index.html:170]: - [ ] Kindly Increase the padding size of the button to match the Figma design. :+1:

File: index.html [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+            cols="30"
+            rows="10"
+            placeholder="input your message here"
+            maxlength="500"
+            required
+          ></textarea>
+            <br/>
+          </div>
+          <button id="submit" type="submit">Get in touch</button>
+          <br />
```
Comment (mahbubul14) [inline index.html:242]: - [ ] Button interactions such as hover, active (when it's pressed), disabled, and enable (default state) should be styled as shown in the Figma template. Kindly work on it. :+1: 

button

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @Kasaija-Kenneth,

Great work on making the changes requested by the previous reviewer 👏

You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "fix bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

- [ ] Please use [CSS Media Queries](https://www.w3schools.com/css/css3_mediaqueries.asp) for your mobile version. It is not currently optimized for the mobile version.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Kasaija-Kenneth 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights
 - Clean and organized code. ✔️
 - Good job following the design. ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**